asmcli/tests/common.sh

```diff
@@ -763,6 +763,12 @@ run_basic_test() {
   label_with_revision "${LT_NAMESPACE}" "${LABEL}"
   roll "${LT_NAMESPACE}"
 
+  if [[ "${EXTRA_FLAGS}" = *--managed* || "${SUBCOMMAND}" = *experimental* ]]; then
+    for _ in {1..5}; do
```

Comment:
Not familiar with the code here. After five retry, how does the test fail?

Reply:
This is WIP. Will update once I am done. Does the 5 retries and `sleep 10` make sense?

Reply:
Yep